SparkPost/2web2ui
# SA-1896: Release Scheduled Reports

### What Changed

- Removed the flag 'allow_scheduled_reports'

### How To Test
 - Use an account without the feature flag.
 - Check that schedule report is an option in the '...' in the view all reports modal. 
 - Check that you can see the scheduled report modal from the /signals/analytics
 - Check that you can edit a report from the  scheduled report modal.
 - Check that you can create a report from the  scheduled report modal.

### To Do

- [ ] Remove feature flag in confluence doc.
